README.md

```diff
@@ -59,30 +59,30 @@ Creating machine...
 Starting local OpenShift cluster...
 OpenShift is available at https://192.168.99.100:8443.
 
-$ oc run hello-minishift --image=gcr.io/google_containers/echoserver:1.4 --port=8080 --expose --service-overrides='{"apiVersion": "v1", "spec": {"type": "NodePort"}}'
-service "hello-minishift" created
-deploymentconfig "hello-minishift" created
+$ oc create -f hello-openshift.json
```

Comment:
Sorry for not looking at this in a while. OK you've convinced me :) Much slicker& easier.

Do you think you could turn that hello-openshift.json into a `new-app`able repo then we could use the oc new-app & follow on the normal openshift flow from there?